File: uva_11069.cpp
# include <iostream>
using namespace std;
int main(void){
    int ans[77];
    ans[1]=1, ans[2]=2, ans[3]=2;
    for (int i=4; i<77; ++i)
        ans[i]=ans[i-2]+ans[i-3];
    int data;
    while(cin >> data)
        cout << ans[data] << endl;
    return 0;
}

File: uva_11069.py
ans=[0 for _ in range(4)]
ans[1], ans[2], ans[3]=1, 2, 2
for i in range(4, 77):
    ans.append(ans[i-2]+ans[i-3])
while True:
    try:
        data=int(input())
        print(ans[data])
    except:
        break